src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/package.json

```
@@ -1,31 +1,27 @@
 {
-  "name": "company.webapplication1",
+  "name": "reactspatest",
```

```suggestion
  "name": "company.webapplication1",
```

"company.webapplication1" is the token used by the template for replacing this.

---

Thanks for spotting this. I missed this one.